tidysic/parser/tree.py

```diff
@@ -29,4 +32,38 @@ def _parse(self) -> None:
             elif AudioFile.is_audio_file(path):
                 self.audio_files.add(AudioFile(path))
             else:
-                self.clutter_files.add(path)
+                self.clutter_files.add(ClutterFile(path))
+        
+        self._parse_clutter_files()
+
+    
+    def _parse_clutter_files(self):
+        """
+        Parse the `ClutterFile` of the `Tree` by assigning them tags shared by
+        all the `AudioFile` of the `Tree`.
+        """
+        tags = self._find_tags_intersection()
+        for clutter_file in self.clutter_files:
+            clutter_file.copy_tags_from(tags)
```

Comment:
Only the root's clutter will get tagged.

Reply:
This is a tree, when you call `self.children.add(Tree(path))` then "the root becomes the children", thus the clutter files from the children will also be tagged.

Reply:
Oh right